### PR TITLE
Add Tooltip & TooltipIcon components

### PR DESCRIPTION
**Pull request by lightness**
### Where

_Add simple Tooltip and TooltipIcon components_

### What

_Components receive tooltip text from props. So it can be used with or without redux_

### Screenshots

image

### CODE REVIEW

File: examples/basic/app.js [hunk 1]
```diff
@@ -296,6 +298,10 @@ const App = function App() {
           </form>
         </div>
 
+        <div className="clearfix">
+          <h3>Tooltip <TooltipIcon text="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum." /></h3>
```
Comment (Pahmurny-itechart) [inline examples/basic/app.js]: Why are you importing both `Tooltip` and `TooltipIcon`, and use only `TooltipIcon` ?

File: lib/components/Tooltip.jsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+  render() {
+    const { for: htmlFor, text, children, className } = this.props;
+    const tooltipClass = classNames(
+      'mdl-tooltip mdl-tooltip--large mdl-tooltip--bottom is-active',
+      className,
+    );
+
+    return (
+      <div className={tooltipClass} htmlFor={htmlFor} style={({ marginLeft: '23px' })}>
```
Comment (Pahmurny-itechart) [inline lib/components/Tooltip.jsx]: *

File: lib/stylesheets/components/_tooltip.scss [hunk 3]
```diff
@@ -0,0 +1,103 @@
+@import "~cw-styleguide/bower_components/material-design-lite/src/tooltip/tooltip";
+
+:root {
+  --min-width-tooltip: 350px;
```
Comment (Pahmurny-itechart) [inline lib/stylesheets/components/_tooltip.scss]: ie11?

File: lib/stylesheets/components/_tooltip.scss [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+    opacity: 1;
+    visibility: visible;
+  }
+
+  // @include media("<=mobile") {
+  //   display: none;
+  // }
+}
+
+@keyframes pulseTopTooltip {
```
Comment (Pahmurny-itechart) [inline lib/stylesheets/components/_tooltip.scss]: Not used

File: lib/stylesheets/components/_tooltip.scss (hunk 3 above)
Comment (lightness) [reply lib/stylesheets/components/_tooltip.scss]: Refactored... Used scss vars instead

File: lib/stylesheets/components/_tooltip.scss (hunk 4 above)
Comment (lightness) [reply lib/stylesheets/components/_tooltip.scss]: Removed

**Comment by lightness** [comment]
In future I plan extend tooltip features:
 - allow show at left/right side
 - totally decouple tooltip and tooltip icon (theoretically tooltip can be applied to any component, not only to TooltipIcon)

But at the moment, this PR blocks https://coverwallet.atlassian.net/browse/C4A-742

**Pull request merged**